platform-includes/configuration/data-to-all-spans/android.mdx

```diff
@@ -0,0 +1,38 @@
+```java
+Sentry.init(options -> {
+    options.setBeforeSendTransaction((transaction, hint) -> {
+        // set the attribute on the root span
+        if (transaction.getContexts().getTrace() == null) {
```

Comment:
Should this be:
```suggestion
        if (transaction.getContexts().getTrace() != null) {
```

Looks like this will cause an NPE. Applies to all samples below.